account_bank_statement_import_camt/models/parser.py

```diff
@@ -173,6 +173,8 @@ def parse_entry(self, node, transaction):
         transaction_base = transaction
         for i, dnode in enumerate(detail_nodes):
             transaction = copy(transaction_base)
+            if transaction.get('unique_import_id') and i > 0:
+                transaction['unique_import_id'] += '-%d' % (i,)
```

Comment:
While very difficult ,before this fix  it  was  still possible for unique_import_id to NOT be unique.
by appending the extra -{number} we ensure uniqueness and avoid raising the SQL uniqueness check...  :+1: